Shaloo/doc linea update 1013 (#1014)
Consensys/doc.linea: docs/get-started/tooling/social-login/arcana.mdx

image: /img/socialCards/arcana.jpg
---

Web3 devs building on the Linea chain can **onboard users** using the [Arcana Auth SDK](https://arcana.network/auth) through the [social login](https://docs.arcana.network/concepts/social-login) feature. 

The SDK also offers an embedded, non-custodial, built-in web3 wallet, the [Arcana wallet](https://docs.arcana.network/concepts/anwallet/). It allows authenticated users to **sign Linea transactions** within the app context. 

## Onboard users

With Arcana Auth SDK's social login, devs can add a single line of code to onboard users through a plug-and-play login UI via `connect()`. Devs can choose one or more of the available [social login providers and custom login options](https://docs.arcana.network/setup/config-auth/) to manage the user onboarding experience. 

<div class="center-container">
  <div class="img-large">
      <img 
            src="/img/get_started/tooling/social_login/arcana/arcana_social_pnp_modal.png"
            alt="Arcana Social Login Plug & Play UI"
      />
  </div>
</div>

> Besides onboarding users, devs can also set up the built-in Arcana wallet to use the preconfigured Linea chain as the default active chain in the wallet. This lets authenticated users to access the Arcana wallet and review, sign Linea transactions without having to manually set up and configure a wallet before using the Linea chain.  

Reviewer: ```suggestion
Besides onboarding users, devs can also set up the built-in Arcana wallet to use the preconfigured Linea chain as the default active chain in the wallet. This lets authenticated users access the Arcana wallet and review and sign Linea transactions without manually setting up and configuring a wallet before using the Linea chain.  
```

To onboard users via social login and enable them to use the Arcana wallet in a web3 app, refer to the [Auth Quickstart Guide](https://docs.arcana.network/quick-start/). 

## Sign Linea transactions

> The Arcana wallet is automatically available for signing transactions once a user logs in to any app integrated with the [Arcana Auth SDK](https://docs.arcana.network/concepts/authsdk). To access the wallet users need to only log in to such an app. They can instantly access the in-app Arcana Wallet and issue transactions or perform supported web3 wallet operations. Users don't need to download, install or configure a wallet for using the Linea chain.

Reviewer: ```suggestion
The Arcana wallet is automatically available for signing transactions once a user logs into any app integrated with the [Arcana Auth SDK](https://docs.arcana.network/concepts/authsdk). To access the wallet users need only log in to such an app. They can instantly access the in-app Arcana Wallet and issue transactions or perform supported web3 wallet operations. Users don't need to download, install, or configure a wallet for using the Linea chain.
```

  - Configure optional Auth SDK usage settings.
3. **Download and install** the appropriate Auth SDK flavor as per the web3 [app type](https://docs.arcana.network/sdk-installation). 
4. **Integrate the app** by using the *client ID* obtained in step 1. Refer to the specific app type [Auth Integration Guide](https://docs.arcana.network/auth/integrate/wagmi/) for details.

:::note

Devs can set up Linea as the [default active chain](https://docs.arcana.network/setup/config-dapp-with-db/) in the wallet. Refer to the dashboard [chain management settings](https://docs.arcana.network/setup/config-wallet-chains/#evm-chains). 

<div class="center-container">
  <div class="img-large">
      <img 
            src="/img/get_started/tooling/social_login/arcana/dashboard_setup.gif"
            alt="Arcana Developer Dashboard"
      />
  </div>
</div>

:::

Devs can also tailor the user experience of signing blockchain transactions on Linea. Besides the look and feel of the wallet, devs can also manage when the wallet UI is displayed within the app context to sign blockchain 
transactions.

[Learn more](https://docs.arcana.network/user-guides/wallet-ui/).

:::note

> The in-app Arcana Wallet supports [JSON/RPC calls and web3 wallet operations](https://docs.arcana.network/auth/web3-ops/evm/).

Reviewer: ```suggestion
The in-app Arcana Wallet supports [JSON-RPC calls and web3 wallet operations](https://docs.arcana.network/auth/web3-ops/evm/).
```

:::

## Wallet access

> Users do not need to download a wallet, or configure it to use the Linea chain manually. To access the wallet users must log in to an app that is integrated with the [Arcana Auth SDK](https://arcana.network/auth).

Reviewer: ```suggestion
Users do not need to download a wallet or manually configure it to use the Linea chain. To access the wallet users must log in to an app that is integrated with the [Arcana Auth SDK](https://arcana.network/auth).
```
